server/views.py

```diff
@@ -179,8 +180,11 @@ def show_all_volunteers(request, page=1):
 
     final_data = paginator.page(page)
 
+    list_pages_before = range(max(1, page - PAGINATION_SHORTCUT_NUMBER), page)
+    list_pages_after = range(page + 1, min(page + PAGINATION_SHORTCUT_NUMBER + 1, paginator.num_pages + 1))
+
     context = {'volunteer_data': final_data, 'availability_now_id': availability_now_id, 'page': page,
-               'num_pages': paginator.num_pages}
+               'num_pages': paginator.num_pages, 'pages_before': list_pages_before, 'pages_after': list_pages_after}
```

Comment:
indentation.
do it like:
context = {
    'foo': 'bar',
    'foo2': 'bar2',
    ....
}

Reply:
well github is removing my spaces, but the fields should be one tab inside

Reply:
Done